packages/ffe-grid-react/src/utils.js

```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore if: I don't see any good way of getting around NODE_ENV */
```

Comment:
I suppose we *could* have two test-suites where one is run (ran?) with NODE_ENV set to "production" but it seems a lot of work for little gain so I'm all for just skipping these.